File: SE4-C/ejer2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE4_C
{
    internal class ejer2
    {
        static void Main(string[] args)
        {
            Console.Write("Año: ");
            int an = int.Parse(Console.ReadLine());

            if((an % 4 == 0 && an%100 != 0) || an%400 ==0)
                Console.WriteLine("\nBisiesto");
            else Console.WriteLine("\nNo bisisesto");

            if (an%2==0) Console.WriteLine("Par");
            else Console.WriteLine("impar");
        }
    }
}

File: SE4-PY/ejer2.py
an = int(input("Año: "))

if((an % 4 == 0 and an%100 != 0) or an%400 ==0):
    print("\nBisiesto")
else: 
    print("\nNo bisisesto")

if (an%2==0): print("Par")
else: print("impar")